src/lib.rs

```diff
@@ -33,15 +38,20 @@ impl PopplerDocument {
     }
     pub fn new_from_data(
         data: &mut [u8],
-        password: &str,
+        password: Option<&str>,
     ) -> Result<PopplerDocument, glib::error::Error> {
         if data.is_empty() {
             return Err(glib::error::Error::new(
                 glib::FileError::Inval,
                 "data is empty",
             ));
         }
-        let pw = CString::new(password).map_err(|_| {
+        let pw = CString::new(if password.is_none() {
+            ""
+        } else {
+            password.expect("That it's not None RIGHT after I check")
```

Comment:
Same here please :)

Reply:
Changed in commit version [6917c0d](https://github.com/DMSrs/poppler-rs/pull/13/commits/6917c0da6d83819b020bf1d7f731caa3d2c136ad) too